### PR TITLE
Add test cases to the MetaDataService of the soul-admin module. (#507)

### PR DESCRIPTION
**Pull request by funpad**
Add test cases to the MetaDataService of the soul-admin module. :D

### CODE REVIEW

File: soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Test cases for MetaDataService.
+ *
+ * @author James Fang (funpad)
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class MetaDataServiceTest {
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java:58]: add final

File: soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+        MetaDataVO dataVo = metaDataService.findById(anyString());
+        // Then
+        Assert.assertEquals(new MetaDataVO(), dataVo);
+
+        // Given
+        final String appName = "appName";
+        MetaDataDO metaDataDO = new MetaDataDO();
+        metaDataDO.setAppName(appName);
+        when(metaDataMapper.selectById(anyString())).thenReturn(metaDataDO);
+        // When
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java:157]: remove comment and empty line

**Review by dengliming — APPROVED** [review]

File: soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java (hunk 2 above)
Comment (funpad) [reply soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java]: Okay, it's done.

File: soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java (hunk 1 above)
Comment (funpad) [reply soul-admin/src/test/java/org/dromara/soul/admin/service/MetaDataServiceTest.java]: Okay, it's done.

**Pull request merged**